soccer/ui/MainWindow.cpp

```diff
@@ -168,6 +169,9 @@ MainWindow::MainWindow(Processor* processor, QWidget* parent)
     if (!_processor->simulation()) {
         _ui.menu_Simulator->setEnabled(false);
     } else {
+        // Pass grCom into the simFieldView
+        _ui.fieldView->setGrCom(grCom);
```

Comment:
Yuck okay.

I really wish this whole file operated off of Context instead of directly calling out to things but oh well.